docs/reference/commands.md

```diff
@@ -1339,7 +1339,7 @@ lakectl branch-protect add <repo uri> <pattern> [flags]
 {:.no_toc}
 
 ```
-lakectl add lakefs://<repository> 'stable_*'
+lakectl branch-protect add lakefs://<repository> 'stable_*'
```

Comment:
These seem unrelated, and I'd rather the author reviewed this.

Reply:
Opened #2976